config/rbac/role.yaml

```diff
@@ -209,3 +209,9 @@ rules:
       - create
       - update
       - delete
+  - apiGroups:
+      - ""
+    resources:
+      - serviceaccounts
+    verbs:
+      - impersonate
```

Comment:
We can skip these; I believe this is use case dependent and should be on user side to define these.

Reply:
Yes 👍 Reverted it as part of `f52f3ab `